### DISCUSSION TITLE
Code scanning and OWASP TOP 10

### DISCUSSION THREAD

**Discussion by dibbdob** [community/community#22752] [New to GitHub]
<p>We are evaluating a move from SonarCloud to GitHub to manage the detection of security vulnerabilities.</p>
<p>At present, we analyse using BitBucket pipelines and push the results to <a href="http://sonarcloud.io" rel="noopener nofollow ugc">sonarcloud.io</a><br>
The results shown in sonarcloud are categorised into a number of areas and one of these is OWASP TOP 10.  This category is extremely useful to us as we have compliance requirements to address issues in this area.</p>
<p>Is there a way that GitHub code scanning can also mark issues as OWASP related?</p>

**Comment by francisfuzz** [comment, marked as answer]
<aside class="quote no-group" data-username="dibbdob" data-post="1" data-topic="136978">
<div class="title">
<div class="quote-controls"></div>
 dibbdob:</div>
<blockquote>
<p>Is there a way that GitHub code scanning can also mark issues as OWASP related?</p>
</blockquote>
</aside>
<p>:wave: hello there and welcome to the GitHub Support Community! There isn’t a way to configure GitHub code scanning to also mark issues as OWASP related today.</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.</p>
<p>Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request? That’s the best place to share requests like these in consideration for future iterations of GitHub features.</p>

**Comment by dibbdob** [comment]
<p>Feedback has now been submitted as requested.</p>
